x-pack/plugins/cases/public/components/create/flyout/use_cases_add_to_new_case_flyout.tsx

```diff
@@ -0,0 +1,45 @@
+/*
```

Comment:
This hook is the direct integration with third party plugins. Plugins will load this hook and use it to open or close the flyout as required.